### DISCUSSION TITLE
Download both 1.5 and inpainting-1.5 model from RunwayML when choosing the 1.5 model

### DISCUSSION THREAD

**Discussion by tar-gezed** [TheLastBen/fast-stable-diffusion#197] [Ideas]
Hi 😃 
First, thanks for this awesome Colab version 😉 

It seems that Automatic1111 version support the dedicated [inpainting model](https://github.com/runwayml/stable-diffusion#inpainting-with-stable-diffusion) by RunwayML.
I thing it would be great to download both model (or at least leave the possibility to the user) when choosing the new 1.5 model on the dropdown from the Colab to let the user switch between the two models during his session (I often do some inpainting when generating new images and if the new inpainting model is better at this I think it's a good idea to have it in this Colab too 😄)

**Comment by TheLastBen** [comment]
Hi, thanks, I'll add that option very soon

**Reply by tar-gezed** [reply]
Perfect, thanks a lot 😃

**Comment by TheLastBen** [comment]
Added https://github.com/TheLastBen/fast-stable-diffusion/commit/8ba38a6b00bf952da2b9c3f735b7613b4a7c8b51
